### DISCUSSION TITLE
Getting Cron to display the default value

### DISCUSSION THREAD

**Discussion by Pete-Fowler** [xrutayisire/react-js-cron#56] [General]
Per the docs, it should be easy to make the component display a default value:
```
       <Cron
            value={cron}
            setValue={onSetCron}
            onError={onError}
        />
```
I inspect the value of cron state in the debugger and it is a valid cron expression, but the value does not display in the component. Am I missing something?

**Comment by xrutayisire** [comment]
Can you give me a code sandbox example of the issue you have?

**Reply by Pete-Fowler** [reply]
Sorry, I was accidentally letting it get a ? for the Quartz format and it worked when I changed that.
